### DISCUSSION TITLE
Docs: example for `sql.expression.Delete` doesn’t seem to fit the section?

### DISCUSSION THREAD

**Discussion by jenstroeger** [sqlalchemy/sqlalchemy#11757] [Usage Questions] [answered]
Hello,

I’m not sure this is intended but the [sqlalchemy.sql.expression.Delete](https://docs.sqlalchemy.org/en/20/core/dml.html#sqlalchemy.sql.expression.Delete) section sports an example [here](https://docs.sqlalchemy.org/en/20/core/dml.html#sqlalchemy.sql.expression.Delete.returning) which uses an `.update()` rather than `.delete()` — or am I missing something?

Much thanks,
Jens

**Comment by zzzeek** [comment, marked as answer]
that method is inherited from the base UpdateBase class, there's a note at the doc linked which mentions this to clarify this is not a method unique to Delete

**Reply by jenstroeger** [reply]
Thanks! I saw that not but wasn’t sure 👍🏼
